### ISSUE TITLE
pushing a new bug to github via git-bug-bridge then pulling back down duplicates the bug

### ISSUE THREAD

**Issue by delta1024** [MichaelMure/git-bug#690]
after setting up a bridge to github using a authtoken (i believe it was option number 3 in bridge-configure) i ran the following commands

    git bug bridge push
    git bug bridge pull
    git bug ls

outputs the following  

    70864ca open	Mirror Select list exits unexpectedly             	Jacob Stannix  	   1 💬
    8009d53 open	program will erase drive with existing partitions…	Jacob Stannix  	   1 💬
    a74b02b open	Mirror Select list exits unexpectedly             	Jacob Stannix  	   1 💬
    5076442 open	program will erase drive with existing partitions…	Jacob Stannix  	   1 💬     

issue only appears when a bug created by git-bug is pushed to github and pulled back down. Does not  occur when the issue is created on github's side

**Comment by MichaelMure** [comment]
Which version are you using?

**Comment by delta1024** [comment]
0.7

**Comment by MichaelMure** [comment]
If you could try with the development version (master) it'd be nice. This is most likely fixed already.

**Comment by delta1024** [comment]
I went and downloaded the latest stable release and installed it. I then went and created a test issue  then configured the bridge.

I've included the commands i used and their output below.

`git bug bridge configure`
[1]: github
[2]: gitlab
[3]: jira
[4]: launchpad-preview
target: 1
name [default]:

Detected projects:
[1]: github.com/delta1024/redbar

[0]: Another project

Select option: 1
Github login: delta1024

[1]: enter my token
[2]: interactive token creation

Existing token for github:
[3]: my_token => (login:delta1024) (14 Jun 21 17:49 MDT)

Select option: 3

Current identity my_token tagged with login delta1024
Successfully configured bridge: default

`git bug bridge push`
new issue: cebb80cb21afbd46e31fabf215c906bec5dde692
exported 1 issues with default bridge

 `git bug bridge pull`
new issue: 7a80bbea84885a15fb4928775c5300b1adc053c6
changed status: 43f9d0a32632278959fcbc31e8133cdbe05474e926675f69ea172d511192701f
imported 1 issues and 0 identities with default bridge
 `git bug ls`
cebb80c open	test bug                                          	Jacob Stannix  	   1 💬
7a80bbe closed	test bug                                          	Jacob Stannix  	   1 💬
